app/views/bulk_updates/show.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="row row-content">
+  <section>
+    <h2>Bulk Update <%= @task_id %></h2>
+
+    <h3>Task Status: <%= @task.fetch('task-status', 'Unknown') %></h3>
+    <h4>Status Message: <%= @task.fetch('status_message', 'Unknown') %></h4>
+
+  </section>
+  <section>
+    <% if @task.fetch('collection-statuses', nil).blank? %>
+      <h4>No Bulk Update Errors for this Task</h4>
+      <!-- TODO put this in table or keep out of table? -->
+    <% else %>
+
+      <h4>Bulk Update Errors</h4>
+      <table class="space-top space-bot" id="bulk-update-errors-table">
+        <thead>
+          <tr>
+            <th>Collection Concept ID</th> <!-- TODO should we be requesting and displaying other info? i.e. entry title, entry id... -->
```

Comment:
I would suggest matching the other tables across the app in display the `entry title`, `short name`, and `version`.